README.md

```diff
@@ -77,24 +96,28 @@ The code for the script you just ran is `index.js`.
 
 Let's go through the skeleton part by part to understand what's going on:
 
-    const daml = require('@da/daml-ledger');
+    const ledger = require('@digital-asset/daml-ledger');
+
+    const daml = ledger.daml;
 
     const uuidv4 = require('uuid/v4');
 
 The first line loads the bindings and allows you to refer to them through the `daml` object.
 
-The second one introduces a dependency that is going to be later used to generate unique identifiers; no need to worry about it now.
+The second one creates a shorthand for the `daml` object in the `daml-ledger` library, that can be used to express DAML values in your code concisely.
```

Comment:
I don't think it's considered bad style to repeat the word `line` instead of `one` here and in the next line.

Reply:
Addressed by https://github.com/digital-asset/ex-tutorial-nodejs/pull/7/commits/b55298166fd9b835812c647937ea52a4fc56a264